### DISCUSSION TITLE
Drag and drop for QFile

### DISCUSSION THREAD

**Discussion by NilGT** [quasarframework/quasar#12625] [General - Components / Directives / etc] [answered]
Hello,
Is there a way to implement a drag and drop for Qfile picker? 
Or as an alternative i could use the QUploader, but i need to be able to use the v-model attribute, since i need to save the user's file
(right now using Vuex to help me with it), then send it later on as a part of a different request,
Would appreciate any ideas,
Thank in advance.

**Comment by MilosPaunovic** [comment, marked as answer]
The `QFile` already supports dropping files, out of the box.
Try to drop something to any example in the docs: [https://quasar.dev/vue-components/file-picker](https://quasar.dev/vue-components/file-picker).

**Reply by NilGT** [reply]
Oh, My bad.
haven't provided feedback in my site so i thought its not working.
also, how can i display the img the user selected?

**Reply by MilosPaunovic** [reply]
Where do you want to show it? More information you provide for your questions will help you get the answers quicker.
